app/views/works/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<% books = @works.where(category: "book") %>
+<% albums = @works.where(category: "album") %>
+<h2>List of Works</h2>
+<h3>Books</h3>
+<table>
+    <tr>
+        <th>Votes</th>
+        <th>Title</th>
+        <th>Created by</th>
+        <th>Published</th>
+        <th>Upvote</th>
+    </tr>
+    <% books.each do |book| %>
+        <tr>
+            <td><%= book.vote_count %></td>
+            <td><%= link_to book.title, work_path(book) %></td>
+            <td><%= book.creator %></td>
+            <td><%= book.publication_year %></td>
+            <td><%= link_to "Upvote", vote_path(book.id) %></td>
+        </tr>
+    <% end %>
+</table>
+<h3>Albums</h3>
+<table>
+    <tr>
+        <th>Votes</th>
+        <th>Title</th>
+        <th>Created by</th>
+        <th>Published</th>
+        <th>Upvote</th>
+    </tr>
+    <% albums.each do |album | %>
+        <tr>
+            <td><%= album.vote_count %></td>
+            <td><%= link_to album.title, work_path(album) %></td>
+            <td><%= album.creator %></td>
+            <td><%= album.publication_year %></td>
+            <td><%= link_to "Upvote", vote_path(album.id) %></td>
+        </tr>
+    <% end %>
```

Comment:
Note that you have three blocks of code that are almost identical. Consider how you can use a partial view with the local variable being the category or a loop to dry up this code.